src/GraphQLParser.ApiTests/GraphQL-Parser.approved.txt

```diff
@@ -455,5 +455,6 @@ namespace GraphQLParser
         STRING = 18,
         COMMENT = 19,
         UNKNOWN = 20,
+        AMPERSAND = 21,
```

Comment:
It is unpleasant that the `UNKNOWN` value ceased to be the last. To save backward compatibility...